File: src/main/java/com/web/Fremdsprache/model/BanchProgressData.java
package com.web.Fremdsprache.model;

import lombok.Builder;
import lombok.Data;

@Builder
public @Data class BanchProgressData {

	int learned_words;
	int detected_words;
	int all_words_dictionary;
	int learning_words;
	long current_level_experience;
	int experience_before_next_level;
	
	byte experience_earned_today;
	short current_level;
	short before_next_level;
	byte how_do_many_days_persistence_of_learning;
}

File: client/src/app/classes/banch-progress-data.ts
export class BanchProgressData {
    
    constructor(
        public learned_words:number,
        public detected_words:number,
        public all_words_dictionary:number,
        public learning_words:number,
        public current_level_experience:number,
        public experience_before_next_level:number,

        public experience_earned_today:number,
        public current_level:number,
        public before_next_level:number,
        public how_do_many_days_persistence_of_learning:number,
        
      ) {}

}
